content/blog/entries/science-museum-implementation/contents.lr

```diff
@@ -0,0 +1,108 @@
+title: Science Museum provider implementation
+---
+categories: 
+
+cc-catalog 
+gsoc
```

Comment:
Please add the gsoc-2020 tag and also add a blog post series. See @makkoncept's PR for an example: https://github.com/creativecommons/creativecommons.github.io-source/pull/282